io/eolearn/tests/test_local_io.py

```diff
@@ -170,6 +179,19 @@ def test_export_import(test_case, test_eopatch):
         ), "Tiff imported into new EOPatch has different dtype as expected"
 
 
+def _execute_with_warning_control(
```

Comment:
since you use this (inlined) in `test_vector_to_raster_result`, would it make sense to move it to test_utils?

Reply:
Hm, not sure. So far we don't have any module for test utils in eo-learn, only in sentinelhub-py, right? So I'm not sure if we should create one only for this.